admin_manual/office/example-docker.rst

```diff
@@ -0,0 +1,118 @@
+================================
+Installation example with Docker
+================================
+
+We'll describe how to get Nextcloud Office running on your server and how to integrate it into your Nextcloud using the docker image Nextcloud and Collabora built.
+
+
+To install it the following dependencies are required:
+
+- A host that can run a Docker container
+- A subdomain or a second domain that the Collabora Online server can run on
+- An Apache server with some enabled modules
+- A valid SSL certificate for the domain that Collabora Online should run on
+- A valid SSL certificate for your Nextcloud
+
+
+1. Install the Collabora Online server
+**************************************
+
+The following steps will download the Collabora Online docker, make sure to replace "cloud.nextcloud.com" with the host that your own Nextcloud runs on. Also make sure to escape all dots with double backslashes (\), since this string will be evaluated as a regular expression (and your bash 'eats' the first backslash.) If you want to use the docker container with more than one Nextcloud, you'll need to use 'domain=cloud\\.nextcloud\\.com\|second\\.nextcloud\\.com' instead. (All hosts are separated by \|.)
```

Comment:
```suggestion
The following steps will download the Collabora Online docker. Make sure to replace "cloud.example.com" with the host that your own Nextcloud runs on. Also make sure to escape all dots with double backslashes (`\\`), since this string will be evaluated as a regular expression (and your bash 'eats' the first backslash.) If you want to use the docker container with more than one Nextcloud, you'll need to use `domain=cloud\\.nextcloud\\.com\|second\\.nextcloud\\.com` instead. (All hosts are separated by `\|`.)
```